app/web_conn.go

```diff
@@ -290,7 +338,14 @@ func (wc *WebConn) readPump() {
 			wc.logSocketErr("websocket.read", err)
 			return
 		}
-		wc.App.Srv().WebSocketRouter.ServeWebSocket(wc, &req)
+
+		// Plugin messages which actions are prefixed with the plugin
+		// prefix should only be dispatched to the plugins
+		if !strings.HasPrefix(req.Action, websocketMessagePluginPrefix) {
+			wc.App.Srv().WebSocketRouter.ServeWebSocket(wc, &req)
+		}
```

Comment:
Added this block to avoid sending to the WS router plugin messages, which would have no handler registered in the server and would log a 500 error